assets/vue/main.js

```diff
@@ -285,7 +285,7 @@ app
     .use(Quasar, quasarUserOptions)
     .use(VueFlatPickr)
     //.use(VuelidatePlugin)
-    .use(vuetify)
+    // .use(vuetify)
```

Comment:
There is a conflict between tailwindcss library and vuetify using clasess like color-primary, this is needed for tailwind to work